src/Components/Tag.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import "../Tag.css";
+import closeIcon from "../MediaFiles/closeicon.svg";
+
+const Tag = ({ word, tagType, removeable, removeTag }) => {
+  const tagStyle = tagType === "field" ? "fieldTag" : "skillTag";
+
+  const handleRemove = () => {
+    event.preventDefault(); // Prevent default button behavior
```

Comment:
?? what is the default behavior of a button? 
what is event in your case? you pass nothing as a parameter